verifier/src/tcb.rs

```diff
@@ -116,17 +137,58 @@ pub struct TcbComponent {
 /// <https://api.trustedservices.intel.com/sgx/certification/v4/tcb?fmspc={}>
 ///
 /// Due to the way the TCB info is signed the contents should be provided as is.
-#[derive(Debug, Deserialize)]
+#[derive(Debug, Deserialize, Clone)]
 #[serde(rename_all = "camelCase")]
-pub struct TcbInfoRaw<'a> {
-    tcb_info: &'a RawValue,
-    signature: &'a str,
+pub struct TcbInfoRaw {
+    tcb_info: Box<RawValue>,
+    #[serde(with = "hex")]
+    signature: Vec<u8>,
```

Comment:
A couple things:
1. Adding the `serde(with`, or perhaps it was just the owned Vec, I started to run into lifetime issues with the `'de` lifetime in the `Deserialize` derive macro.  I struggled a bit and finally punted to a `Box<RawValue>`
2. I updated this to serde with, so that it would fail to parse to hex, sooner rather than later. I wanted to for the issue and next update `DateTime` but only the `chrono` versions which don't support `PartialOrd` were available to derive.  We could use chrono and then compare durations if we wanted...

Reply:
>  Adding the serde(with, or perhaps it was just the owned Vec, I started to run into lifetime issues with the 'de lifetime in the Deserialize derive macro. I struggled a bit and finally punted to a Box<RawValue>

Sometimes this can be fixed with implementing `DeserializeOwned` instead?

Reply:
The `Box<RawValue>` seems a bit funky, I want to play with this a bit myself, maybe we can make it a little cleaner

Reply:
This should be addressed in https://github.com/mobilecoinfoundation/attestation/pull/85/commits/e60468596073a3bbf20e7536a3d9b1b91955ac2b, by using the [`#[serde(borrow)]`](https://serde.rs/lifetimes.html#borrowing-data-in-a-derived-impl) attribute.